DataProviders/GoogleCredentialProvider.py

```diff
@@ -0,0 +1,21 @@
+import configparser
+import gspread
+from oauth2client.service_account import ServiceAccountCredentials
+
+
+# TODO: Should this a singleton-patterned accessible class to make this more robust?
+
+__scope = ["https://spreadsheets.google.com/feeds",
+           "https://www.googleapis.com/auth/drive"]
+
+config = configparser.ConfigParser()
```

Comment:
If you're gonna make all of this stuff private you might as well make config private, too.